### ISSUE TITLE
Add swipe to delete conversations

### ISSUE THREAD

**Issue by pR0Ps** [SilenceIM/Silence#564]
Implementation possibilities:

 1. Swipe from the right (opposite of archive, consistent with standard iOS behavior)

2. Option to change the existing archive action to delete

I lean towards option 1

**Comment by atao** [comment]
Good idea, the same fonctionnality exist in Google Inbox. Very usefull!

**Comment by OhSoGood** [comment]
+1
I terribly miss that feature, it's so useful
